rust/lance-table/src/feature_flags.rs

```diff
@@ -14,6 +14,11 @@ pub const FLAG_DELETION_FILES: u64 = 1;
 /// Row ids are table after moves, but not updates. Fragments contain an index
 /// mapping row ids to row addresses.
 pub const FLAG_MOVE_STABLE_ROW_IDS: u64 = 2;
+/// Files are written with the new v2 format (temporary flag, will be removed
+/// once v2 is the default format)
+pub const FLAG_USE_V2_FORMAT: u64 = 4;
+/// The first bit that is invalid as a feature flag
+pub const FLAG_INVALID: u64 = 8;
```

Comment:
Technically, not really invalid, but rather unknown.

Reply:
Changed to `FLAG_UNKNOWN`.